plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt

```diff
@@ -382,6 +382,7 @@ open class KtlintPlugin : Plugin<Project> {
             description = "Format all .kts files in project directory " +
                 "to ensure that they are formatted according to ktlint."
             configurePluginTask(target, extension, ktLintConfig) {
+                android.set(false)
```

Comment:
Why would someone want to not use the android format for their scripts?
Shouldn't we just honor the flag universally?

Reply:
Good question :thinking: My opinion was somehow biased that android style guide should be only applied for android project sources, but not for kotlin script files.

I would remove that.

Reply:
I think I agree that removing it and going with the universal configuration is better. Alternatively, offer a separate configuration flag.

Reply:
Removed this.